ceph/nvmeof/initiators/linux/nvme_cli.py

```diff
@@ -89,7 +89,7 @@ def list_subsys(self, **kwargs):
             cmd=f"nvme list-subsys {device} {config_dict_to_string(kwargs)}", sudo=True
         )
 
-    def list_spdk_drives(self):
+    def list_spdk_drives(self, nsid_device_pair=None):
         """List the NVMe Targets only SPDK drives.
 
```

Comment:
When and why does a user need to pass nsid_device_pair and what happens when its passed or left at None.

Reply:
This has to be passed when we need device name and NSID whcih are mandatory parameters for reservation cmds 
if it is None this would be executed the usual way we get devices today, when passed we get device and NSID pair